core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala

```diff
@@ -0,0 +1,56 @@
+package scala.spores
+package neg
+package basic
+
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.Test
+
+import util._
+
+@RunWith(classOf[JUnit4])
+class AkkaLikeFutureNeg {
+
+  @Test
+  def `wrong shape, incorrect val def list`() {
+    expectError("invalid reference to object") {
+      """
+      import scala.spores._
+      import scala.concurrent.Future
+
+      case class Request[T](value: T)
+      case class Response[T](value: T)
+
+      trait ActorSender[T] {
+        def send(r: Response[T]): Future[T]
+      }
+
+      object ConcreteSender extends ActorSender[String] {
+        def send[T](r: Response[T]): Future[T] = ???
```

Comment:
Returning a future here is a bit surprising, and unlike Akka, so people reading the example might get tripped up by this. ;-)